worldedit-core/src/main/java/com/sk89q/worldedit/regions/factory/CylinderRegionFactory.java

```diff
@@ -26,12 +26,16 @@
 
 public class CylinderRegionFactory implements RegionFactory {
 
-    private final double height;
+    private double height;
 
     public CylinderRegionFactory(double height) {
         this.height = height;
     }
 
+    public void setHeight(double height) {
```

Comment:
Given that this is the only parameter, instead of making this mutable, you could just construct a new one in the brush code?